tests/data/miscellaneous/setitem_getitem.py

```diff
@@ -0,0 +1,52 @@
+x[1]
+x[1,]
+x[(1,)]
+x[1, 2]
+x[1, 2,]
+x[(1, 2)]
+x[1, 2]
+x[(1, 2)]
+x[(1, 2),]
+
+x[1] = 1
+x[1,] = 1
+x[(1,)] = 1
+x[1, 2] = 1
+x[1, 2,] = 1
+x[(1, 2)] = 1
+x[1, 2] = 1
+x[(1, 2)] = 1
+x[(1, 2),] = 1
```

Comment:
Add some test cases that use `del x[(1)]` and similar, just because it's different in the grammar